core-web/libs/block-editor/src/lib/extensions/bubble-menu/dot-bubble-menu.extension.ts

```diff
@@ -31,6 +31,7 @@ export function DotBubbleMenuExtension(viewContainerRef: ViewContainerRef) {
     const changeToElement = changeToComponent.location.nativeElement;
 
     return BubbleMenu.extend<BubbleMenuOptions>({
+        name: 'bubbleMenu',
```

Comment:
@rjvelazco turns out if you don't add name to a extension tiptap add "extension" as default and when you have more than one it shows a "duplicated extension" warn in the console.